package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Intercept imports in Node.js",
   "main": "index.js",
   "scripts": {
-    "test": "c8 --check-coverage --lines 90 imhotap --runner test/runtest --files test/{hook,low-level,other}/*",
+    "test": "c8 --check-coverage --lines 80 imhotap --runner test/runtest --files test/{hook,low-level,other,typescript}/*",
```

Comment:
Why would the coverage drop? The new code only adds a few functions that should be called by the test, so if anything it should _increase_ the coverage. In any case, if that doesn't work feel free to revert to the previous approach, all I was really asking was adding more Node versions if at all possible, but otherwise we can figure out coverage later.

Reply:
I saw some issues about the wrong coverage being reported by c8 but not sure if this is one of them.
Anyways, I separated ts tests from js tests again but still run them the same way as js tests (without enforcing 
 a minimum coverage).